.github/README.md

```diff
@@ -79,6 +79,12 @@ AstroNvim is an aesthetic and feature-rich neovim config that is extensible and
 mv ~/.config/nvim ~/.config/nvimbackup
 ```
 
+### Remove cache from the previous installation
+
+```
+rm -rf ~/.local/share/nvim
```

Comment:
Could you please change this to `mv ~/.local/share/nvim ~/.local/share/nvim.bak` just so we aren't having users blindly deleting directories if they aren't necessarily paying attention?

Reply:
done